pkg/handler/iam_handler.go

```diff
@@ -106,11 +109,29 @@ func NewIAMHandler(ctx context.Context, organizationsClient, foldersClient, proj
 	return h, nil
 }
 
+// Cleanup removes expired IAM bindings added by AOD from the IAM policies of the resources in the request.
+func (h *IAMHandler) Cleanup(ctx context.Context, r *v1alpha1.IAMRequest) (nps []*v1alpha1.IAMResponse, retErr error) {
```

Comment:
Though we have talked about this, but I wonder what will be the best approach here @yolocs :
1. when cleanup triggered on a request, cleanups all expired the AOD binding in the policy. 
2. only cleanups bindings on the request that are expired.
3. only cleanups bindings on the request no matter if it is expired.

Reply:
I think it should be a combination of 1 and 3. It's safe to clean up all expired bindings. For 3, worst case scenario, a user needs to reopen the PR or open a new one.

Reply:
Updated.